test/functional/clipboard/clipboard_provider_spec.lua

```diff
@@ -62,7 +63,7 @@ local function basic_register_test()
 end
 
 describe('clipboard usage', function()
-  setup(reset)
+  before_first(reset)
```

Comment:
How is this different than `setup()`, which only runs once per `describe()` block?